cpu/samd21/syscalls.c

```diff
@@ -115,18 +116,24 @@ void _exit(int n)
  *
  * The current heap implementation is very rudimentary, it is only able to allocate
  * memory. But it does not
- * - check if the returned address is valid (no check if the memory very exists)
  * - have any means to free memory again
  *
- * TODO: check if the requested memory is really available
- *
  * @return [description]
```

Comment:
copied it verbatim from http://pubs.opengroup.org/onlinepubs/7908799/xsh/brk.html